kcalcore/icaltimezones.cpp

```diff
@@ -1313,10 +1313,10 @@ QList<QDateTime> ICalTimeZoneSourcePrivate::parsePhase( icalcomponent *c,
   }
 
   // Convert DTSTART to QDateTime, and from local time to UTC
-  const QDateTime localStart = toQDateTime( dtstart );   // local time
   dtstart.second -= prevOffset;
   dtstart.is_utc = 1;
   const QDateTime utcStart = toQDateTime( icaltime_normalize( dtstart ) );   // UTC
+  const QDateTime localStart = utcStart.toLocalTime();  // local time
```

Comment:
Hrm, this may convert the utcStart into localStart where localStart is in the current SYSTEM local time; which may not be what is required here... the original code may have been intended to find the localStart in the timezone specified by the VCALENDAR entry VTIMEZONE section, perhaps?